tests/ops/test_moba.py

```
@@ -0,0 +1,168 @@
+    return flash_attn_varlen_func(
+        q.squeeze(0), k.squeeze(0), v.squeeze(0),
+        cu_seqlens_q=cu_seqlens,
+        cu_seqlens_k=cu_seqlens,
+        max_seqlen_q=max_seqlen,
+        max_seqlen_k=max_seqlen,
+        causal=True,
+    ).unsqueeze(0)
+
+
+# When `topk - 1` is large enough to cover every earlier chunk, MoBA's
+# per-query attention set equals (chunk-local causal prefix) ∪ (all earlier
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF003)